lambdas/answer/src/main/java/uk/gov/di/ipv/cri/kbv/api/handler/QuestionAnswerHandler.java

```diff
@@ -50,19 +50,18 @@
     private static final String ERROR_KEY = "error";
     private static final String LAMBDA_NAME = "post_answer";
     private final ObjectMapper objectMapper;
-    private final KBVService kbvService;
-    private final KBVStorageService kbvStorageService;
+    private KBVService kbvService;
+    private KBVStorageService kbvStorageService;
```

Comment:
could this handler use the ServiceRegistry in the same way as QuestionHandler to produce a KBVService?

Reply:
Yes, the ServiceFactory should be used here too